nixos/modules/security/krb5/krb5-conf-format.nix

```
@@ -62,15 +62,24 @@ rec {
           };
           access = mkOption {
             type = either (listOf (enum [
+              "all"
               "add"
               "cpw"
               "delete"
+              "get-keys"
               "get"
               "list"
               "modify"
             ])) (enum [ "all" ]);
```

```suggestion
            type = coercedTo str singleton (listOf (enum [
              "all"
              "add"
              "cpw"
              "delete"
              "get-keys"
              "get"
              "list"
              "modify"
            ]));
```

This is getting a mess with "all" being in two places now, I would write this or something similar and then have an assertion that "all" isn't combinable with the others except for get-keys.

---

I think I will write an assertion that checks for any combination involving "all" and anything else that is not "get-keys". I feel it's the simplest and clearest option.